src/mapnik_color.cpp

```diff
@@ -183,10 +183,11 @@ v8::Local<v8::Value> Color::NewInstance(mapnik::color const& color) {
     Color* c = new Color();
     c->this_ = std::make_shared<mapnik::color>(color);
     v8::Local<v8::Value> ext = Nan::New<v8::External>(c);
-    return scope.Escape(Nan::New(constructor)->GetFunction()->NewInstance(1, &ext));
+    v8::MaybeLocal<v8::Object> maybe_local = Nan::NewInstance(Nan::New(constructor)->GetFunction(), 1, &ext);
```

Comment:
`v8::MaybeLocal` has been around at least since node v4.8, so I guess this is not part of the compatibility changes for node 10 (?). I understand that it is always good to keep up with upstream, but I was wondering: what was your main motivation to cherry-pick this change?

Reply:
It's not about maybe_local, but about `->NewInstance` in this case. It changed the API in node 10, so to keep compatibility Nan is used. Upstream had done the change with `maybe_local` in mind, but changing both so I picked their change and add the Nan::NewInstance in the function they've missed.

Reply:
:+1: